doc/manual/flashing.rst

```diff
@@ -4,6 +4,47 @@
 .. note::
   If you have purchased a pre-assembled system from M-Labs or QUARTIQ, the gateware and firmware of your devices will already be flashed to the newest version of ARTIQ. Flashing your device is only necessary if you obtained your hardware in a different way, or if you want to change your system configuration or upgrade your ARTIQ version after the original purchase. Otherwise, skip straight to :doc:`configuring`.
 
+.. _types-of-boards:
```

Comment:
This isn't the correct location for this information -- IMO if anywhere it should be on the core device page, but it's primarily obtrusive here, where system description files aren't relevant and the distinction between artiq and artiq-zynq is soon not to be (#2559).